From a8c3f5121fac2cc4f8e3221a5a06b63d2e9c6f02 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 27 Sep 2013 00:05:07 +0200
Subject: [PATCH] man: drop references to "cgroup" wher appropriate

Since cgroups are mostly now an implementation detail of systemd lets
deemphasize it a bit in the man pages. This renames systemd.cgroup(5) to
systemd.resource-control(5) and uses the term "resource control" rather
than "cgroup" where appropriate.

This leaves the word "cgroup" in at a couple of places though, like for
example systemd-cgtop and systemd-cgls where cgroup stuff is at the core
of what is happening.

Conflicts:
	man/systemd.exec.xml
---
 Makefile-man.am                                    |  2 +-
 man/loginctl.xml                                   |  4 ++--
 man/machinectl.xml                                 |  4 ++--
 man/pam_systemd.xml                                |  2 +-
 man/systemctl.xml                                  | 14 ++++++-------
 man/systemd-cgls.xml                               |  6 +++---
 man/systemd-cgtop.xml                              |  4 ++--
 man/systemd.exec.xml                               |  5 +++--
 man/systemd.mount.xml                              |  6 +++---
 ...emd.cgroup.xml => systemd.resource-control.xml} | 24 ++++++++++------------
 man/systemd.scope.xml                              |  2 +-
 man/systemd.service.xml                            |  6 +++---
 man/systemd.slice.xml                              | 22 ++++++++++----------
 man/systemd.socket.xml                             | 12 +++++------
 man/systemd.swap.xml                               |  6 +++---
 15 files changed, 59 insertions(+), 60 deletions(-)
 rename man/{systemd.cgroup.xml => systemd.resource-control.xml} (96%)

diff --git a/Makefile-man.am b/Makefile-man.am
index 1f63595..c8a4342 100644
--- a/Makefile-man.am
+++ b/Makefile-man.am
@@ -78,7 +78,6 @@ MANPAGES += \
 	man/systemd-update-utmp.service.8 \
 	man/systemd.1 \
 	man/systemd.automount.5 \
-	man/systemd.cgroup.5 \
 	man/systemd.device.5 \
 	man/systemd.exec.5 \
 	man/systemd.journal-fields.7 \
@@ -86,6 +85,7 @@ MANPAGES += \
 	man/systemd.mount.5 \
 	man/systemd.path.5 \
 	man/systemd.preset.5 \
+	man/systemd.resource-control.5 \
 	man/systemd.scope.5 \
 	man/systemd.service.5 \
 	man/systemd.slice.5 \
diff --git a/man/loginctl.xml b/man/loginctl.xml
index ef0dfc0..1b54ff7 100644
--- a/man/loginctl.xml
+++ b/man/loginctl.xml
@@ -117,8 +117,8 @@
                                 <term><option>-l</option></term>
                                 <term><option>--full</option></term>
 
-                                <listitem><para>Do not ellipsize cgroup
-                                members.</para>
+                                <listitem><para>Do not ellipsize
+                                process tree entries.</para>
                                 </listitem>
                         </varlistentry>
 
diff --git a/man/machinectl.xml b/man/machinectl.xml
index 5efa3a5..2ed9f2e 100644
--- a/man/machinectl.xml
+++ b/man/machinectl.xml
@@ -117,8 +117,8 @@
                                 <term><option>-l</option></term>
                                 <term><option>--full</option></term>
 
-                                <listitem><para>Do not ellipsize cgroup
-                                members.</para>
+                                <listitem><para>Do not ellipsize
+                                process tree entries.</para>
                                 </listitem>
                         </varlistentry>
 
diff --git a/man/pam_systemd.xml b/man/pam_systemd.xml
index 951ae20..0e25a4a 100644
--- a/man/pam_systemd.xml
+++ b/man/pam_systemd.xml
@@ -55,7 +55,7 @@
                 <title>Description</title>
 
                 <para><command>pam_systemd</command> registers user
-                sessions in the systemd login manager
+                sessions with the systemd login manager
                 <citerefentry><refentrytitle>systemd-logind.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
                 and hence the systemd control group hierarchy.</para>
 
diff --git a/man/systemctl.xml b/man/systemctl.xml
index 51c34b7..a70c964 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -187,7 +187,7 @@ systemctl start foo
         <term><option>--full</option></term>
 
         <listitem>
-          <para>Do not ellipsize unit names, cgroup members, and
+          <para>Do not ellipsize unit names, process tree entries, and
           truncate unit descriptions in the output of
           <command>list-units</command> and
           <command>list-jobs</command>.</para>
@@ -724,14 +724,14 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
           <listitem>
             <para>Set the specified unit properties at runtime where
             this is supported. This allows changing configuration
-            parameter properties such as resource management controls at
+            parameter properties such as resource control settings at
             runtime. Not all properties may be changed at runtime, but
-            many resource management settings (primarily those in
-            <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>)
+            many resource control settings (primarily those in
+            <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>)
             may. The changes are applied instantly, and stored on disk
             for future boots, unless <option>--runtime</option> is
-            passed, in which case the settings only apply until the next
-            reboot. The syntax of the property assignment follows
+            passed, in which case the settings only apply until the
+            next reboot. The syntax of the property assignment follows
             closely the syntax of assignments in unit files.</para>
 
             <para>Example: <command>systemctl set-property foobar.service CPUShares=777</command></para>
@@ -1299,7 +1299,7 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
       <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>loginctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>systemd.resource-management</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>wall</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry>
diff --git a/man/systemd-cgls.xml b/man/systemd-cgls.xml
index fcaf894..432706b 100644
--- a/man/systemd-cgls.xml
+++ b/man/systemd-cgls.xml
@@ -116,8 +116,8 @@
                                 <term><option>-l</option></term>
                                 <term><option>--full</option></term>
 
-                                <listitem><para>Do not ellipsize cgroup
-                                members.</para>
+                                <listitem><para>Do not ellipsize
+                                process tree members.</para>
                                 </listitem>
                         </varlistentry>
 
@@ -132,7 +132,7 @@
                                 <term><option>-m <replaceable>MACHINE</replaceable></option></term>
                                 <term><option>--machine=<replaceable>MACHINE</replaceable></option></term>
 
-                                <listitem><para>Limit cgroups shown to
+                                <listitem><para>Limit control groups shown to
                                 the part corresponding to the
                                 container <replaceable>MACHINE</replaceable>.
                                 </para></listitem>
diff --git a/man/systemd-cgtop.xml b/man/systemd-cgtop.xml
index 51549c4..7faedfb 100644
--- a/man/systemd-cgtop.xml
+++ b/man/systemd-cgtop.xml
@@ -80,7 +80,7 @@
                 <varname>MemoryAccounting=1</varname> and
                 <varname>BlockIOAccounting=1</varname> settings in the
                 unit files in question (See
-                <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+                <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>
                 for details).</para>
 
                 <para>To emphasize this: unless
@@ -274,7 +274,7 @@
                         <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>systemd-cgls</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-                        <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                        <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                         <citerefentry><refentrytitle>top</refentrytitle><manvolnum>1</manvolnum></citerefentry>
                 </para>
         </refsect1>
diff --git a/man/systemd.exec.xml b/man/systemd.exec.xml
index 5721dc1..28d7fd4 100644
--- a/man/systemd.exec.xml
+++ b/man/systemd.exec.xml
@@ -1017,8 +1017,9 @@
                           <citerefentry><refentrytitle>systemd.swap</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.mount</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd.directives</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+                          <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>systemd.directives</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>exec</refentrytitle><manvolnum>3</manvolnum></citerefentry>
                   </para>
         </refsect1>
 
diff --git a/man/systemd.mount.xml b/man/systemd.mount.xml
index e5b5c3c..48af1ca 100644
--- a/man/systemd.mount.xml
+++ b/man/systemd.mount.xml
@@ -76,8 +76,8 @@
                 <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                 which define the way the processes are terminated, and
                 in
-                <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                which configure control group settings for the
+                <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                which configure resource control settings for the
                 processes of the service. Note that the User= and
                 Group= options are not particularly useful for mount
                 units specifying a <literal>Type=</literal> option or
@@ -302,7 +302,7 @@
                           <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.device</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>proc</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
diff --git a/man/systemd.cgroup.xml b/man/systemd.resource-control.xml
similarity index 96%
rename from man/systemd.cgroup.xml
rename to man/systemd.resource-control.xml
index cc0eb15..2427f208 100644
--- a/man/systemd.cgroup.xml
+++ b/man/systemd.resource-control.xml
@@ -22,9 +22,9 @@ You should have received a copy of the GNU Lesser General Public License
 along with systemd; If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<refentry id="systemd.cgroup">
+<refentry id="systemd.resource-control">
   <refentryinfo>
-    <title>systemd.cgroup</title>
+    <title>systemd.resource-control</title>
     <productname>systemd</productname>
 
     <authorgroup>
@@ -38,13 +38,13 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
   </refentryinfo>
 
   <refmeta>
-    <refentrytitle>systemd.cgroup</refentrytitle>
+    <refentrytitle>systemd.resource-control</refentrytitle>
     <manvolnum>5</manvolnum>
   </refmeta>
 
   <refnamediv>
-    <refname>systemd.cgroup</refname>
-    <refpurpose>Control Group configuration unit settings</refpurpose>
+    <refname>systemd.resource-control</refname>
+    <refpurpose>Resource control unit settings</refpurpose>
   </refnamediv>
 
   <refsynopsisdiv>
@@ -63,12 +63,10 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
     <para>Unit configuration files for services, slices, scopes,
     sockets, mount points, and swap devices share a subset of
-    configuration options which configure the control group settings
-    for spawned processes.</para>
-
-    <para>Control Groups is a concept for organizing processes in a
-    hierarch tree of named groups for the purpose of resource
-    management.</para>
+    configuration options for resource control of spawned
+    processes. Internally, this relies on the the Control Groups
+    kernel concept for organizing processes in a hierarchial tree of
+    named groups for the purpose of resource management.</para>
 
     <para>This man page lists the configuration options shared by
     those six unit types. See
@@ -82,7 +80,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     and
     <citerefentry><refentrytitle>systemd.swap</refentrytitle><manvolnum>5</manvolnum></citerefentry>
     for more information on the specific unit configuration files. The
-    execution-specific configuration options are configured in the
+    resource control configuration options are configured in the
     [Slice], [Scope], [Service], [Socket], [Mount], or [Swap]
     sections, depending on the unit type.</para>
   </refsect1>
@@ -91,7 +89,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <title>Options</title>
 
     <para>Units of the types listed above can have settings
-    for cgroup configuration:</para>
+    for resource control configuration:</para>
 
     <variablelist class='unit-directives'>
 
diff --git a/man/systemd.scope.xml b/man/systemd.scope.xml
index e8fa1ae..9813e0a 100644
--- a/man/systemd.scope.xml
+++ b/man/systemd.scope.xml
@@ -86,7 +86,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
       <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd-run</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.directives</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
     </para>
diff --git a/man/systemd.service.xml b/man/systemd.service.xml
index 8eda85f..45f96fa 100644
--- a/man/systemd.service.xml
+++ b/man/systemd.service.xml
@@ -76,8 +76,8 @@
                 <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                 which define the way the processes of the service are
                 terminated, and in
-                <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                which configure control group settings for the
+                <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                which configure resource control settings for the
                 processes of the service.</para>
 
                 <para>Unless <varname>DefaultDependencies=</varname>
@@ -1006,7 +1006,7 @@
                           <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.directives</refentrytitle><manvolnum>7</manvolnum></citerefentry>
                   </para>
diff --git a/man/systemd.slice.xml b/man/systemd.slice.xml
index b7b0622..4d27ddf 100644
--- a/man/systemd.slice.xml
+++ b/man/systemd.slice.xml
@@ -58,14 +58,14 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <literal>.slice</literal> encodes information about a slice which
     is a concept for hierarchially managing resources of a group of
     processes. This management is performed by creating a node in the
-    control group tree. Units that manage processes (primarilly scope
-    and service units) may be assigned to a specific slice. For each
-    slice, certain resource limits may the be set that apply to all
-    processes of all units contained in that slice. Slices are
-    organized hierarchially in a tree. The name of the slice encodes
-    the location in the tree. The name consists of a dash-separated
-    series of names, which describes the path to the slice from the
-    root slice. The root slice is named,
+    Linux Control Group (cgroup) tree. Units that manage processes
+    (primarilly scope and service units) may be assigned to a specific
+    slice. For each slice, certain resource limits may the be set that
+    apply to all processes of all units contained in that
+    slice. Slices are organized hierarchially in a tree. The name of
+    the slice encodes the location in the tree. The name consists of a
+    dash-separated series of names, which describes the path to the
+    slice from the root slice. The root slice is named,
     <filename>-.slice</filename>. Example:
     <filename>foo-bar.slice</filename> is a slice that is located
     within <filename>foo.slice</filename>, which in turn is located in
@@ -89,9 +89,9 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     files. The common configuration items are configured
     in the generic [Unit] and [Install] sections. The
     slice specific configuration options are configured in
-    the [Slice] section. Currently, only generic cgroup settings
+    the [Slice] section. Currently, only generic resource control settings
     as described in
-    <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>7</manvolnum></citerefentry> are allowed.
+    <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>7</manvolnum></citerefentry> are allowed.
     </para>
 
     <para>Unless <varname>DefaultDependencies=false</varname>
@@ -110,7 +110,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <para>
       <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-      <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+      <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.scope</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
       <citerefentry><refentrytitle>systemd.special</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
diff --git a/man/systemd.socket.xml b/man/systemd.socket.xml
index 419a38c..8c88d9f 100644
--- a/man/systemd.socket.xml
+++ b/man/systemd.socket.xml
@@ -78,11 +78,11 @@
                 <option>ExecStopPost=</option> commands are executed
                 in, and in
                 <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                which define the way the processes are
-                terminated, and in
-                <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                which configure control group settings for the
-                processes of the service.</para>
+                which define the way the processes are terminated, and
+                in
+                <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                which configure resource control settings for the
+                processes of the socket.</para>
 
                 <para>For each socket file a matching service file
                 (see
@@ -712,7 +712,7 @@
                           <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.directives</refentrytitle><manvolnum>7</manvolnum></citerefentry>
                   </para>
diff --git a/man/systemd.swap.xml b/man/systemd.swap.xml
index 18ef5b0..813ae6c 100644
--- a/man/systemd.swap.xml
+++ b/man/systemd.swap.xml
@@ -76,8 +76,8 @@
                 <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                 which define the way the processes are
                 terminated, and in
-                <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                which configure control group settings for the
+                <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                which configure resource control settings for the
                 processes of the service.</para>
 
                 <para>Swap units must be named after the devices
@@ -206,7 +206,7 @@
                           <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.kill</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-                          <citerefentry><refentrytitle>systemd.cgroup</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+                          <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.device</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>systemd.mount</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
                           <citerefentry><refentrytitle>swapon</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
